packages/worker/src/worker-options.ts

```diff
@@ -408,14 +408,6 @@ export interface ReplayWorkerOptions
    *  @default workflow name from given history
    */
   replayName?: string;
```

Comment:
Is it really useful to expose this config? Also, default comment is wrong (actual value: `(options.replayName ?? 'fake_replay_queue') + '-' + this.replayWorkerCount`).

Reply:
Core used to require this value, some users are setting it so I didn't want to remove.

Reply:
We may deprecate it though. Or otherwise, we should rewrite the associated doc so that users understand why/when/what they should provide.